registry/data-models/models.py

```diff
@@ -0,0 +1,168 @@
+                                  +-----------------+
+                                          |  
+                                          |
+                                          | 
+                            +------------------------------+    
+                            |                              |      
+                            | 1:n                          | 1:n         
+                  +------------------+           +------------------+
+      +---------- |   FeatureName    |           |     Anchor       |
+      |      1:n  +------------------+           +------------------+
+      |                 ｜                             ｜          |
+      |                 ｜ 1:n                         ｜1:n       |
+      |                 +------------------------------+          |
+      |                               ｜                           |
+      |                               ｜1:n                       \|/ has
+      |   +--------------+    +---------------+                 +----------------+
+      |   |Transformation|----|    Feature    | --------------- |   DataSource   |            
+      |   +--------------+ has+---------------+                 +----------------+
+      |                              /|\   |                     /|\           |extends
+      |                               |    |                      |            |
+      |                               |    +----------------------|---------+  |
+      |                               |                           |         |  |
+      |          +--------------------------------------+         |      has| \|/
+      |          |                                      |         |        +----------+
+      |          |extends                               |extends  |has     |  Source  |
+      |    +----------------+                      +-----------------+     +----------+
+      |    | DerivedFeature |                      |  AnchorFeature  |          /|\
+      |    +----------------+                      +-----------------+           |
+      |            |                                                             |
+      |            |                                                             |extends
+      |            |                                                       +------------------+
+      |            +-------------------------------------------------------|   FeatureSource  |
+      |                                                                has +------------------+
+      |                                                                             |
+      +-----------------------------------------------------------------------------|
+                      
+                      
+                      
+                      
+                      
+                      
+"""
+
+
+class FeatureId(BaseModel):
+    """
+    Id for Feature, it's unique ID represents Feature.
+    """
+    pass
```

Comment:
should we add str id fields for all of these pre-emptively?

Reply:
Added str id field.